string_array.go

```diff
@@ -1,12 +1,34 @@
 package main
 
 import (
+	"flag"
+	"log"
 	"strings"
 )
 
+var _ flag.Getter = StringArrayGetter{}
+
 type StringArray []string
 
+type StringArrayGetter struct {
+	stringArray *StringArray
+}
+
+func (a StringArrayGetter) Get() interface{} {
+	log.Printf("vracim picoviny: %v", a.stringArray)
+	return []string(*a.stringArray)
+}
+
+func (a StringArrayGetter) Set(s string) error {
+	return a.stringArray.Set(s)
+}
+
+func (a StringArrayGetter) String() string {
+	return a.stringArray.String()
+}
+
 func (a *StringArray) Set(s string) error {
+	log.Printf("apenduju picoviny: %s", s)
```

Comment:
I don't know that this means, but it looks like it shouldn't be here :)

Reply:
oops